ci/arm/gitlab-ci.yml

```diff
@@ -0,0 +1,10 @@
+linux-arm64:
+  script:
+    - uname -a
```

Comment:
Remove these before merging?

Reply:
no, this is because the logs dont have this data, and when you read the logs it is quite nice to have